### PR TITLE
3.11.0

### PR DESCRIPTION
**Pull request by trentm**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -181,12 +181,10 @@ If you have access to make releases, the process is as follows:
         ```
     - Add missing changelog entries, if any. (Typically commits will include changelog entries in the "Unreleased" section.)
 1. If a major or minor release, update the EOL table in `docs/upgrading.asciidoc`. EOL is 18 months after release date.
-1. Commit changes with message `x.y.z` where `x.y.z` is the version in `package.json`
-1. Tag the commit with `git tag vx.y.x`, for example `git tag v1.2.3`
-1. Reset the latest major branch (currently `3.x`) to point to the current master, e.g. `git branch -f 3.x master`
-1. Run tests with `npm test`
-1. Push commits and tags upstream with `git push upstream master && git push upstream --tags` (and optionally to your own fork as well)
-1. Update the latest major branch on upstream with `git push upstream <major_branch>`
+1. Make a PR titled `x.y.z` -- where `x.y.z` is the new version in package.json -- with these changes.
```
Comment (astorm) [inline CONTRIBUTING.md:184]: 👍 for writing this down

Comment (astorm) [inline CONTRIBUTING.md:184]: 👍 for writing this down

**Review by astorm — APPROVED** [review]
Looks good from here -- I see a CHANGELOG that's had its unreleased section updated, this new version added to the EOL table, and both the agent version and the elastic-apm-http-client dependency version updated, and a adjustment to the release instructions that reflects our evolving reality. 

Approved.

**Comment by trentm** [comment]
elastic-apm-node@3.11.0 published

**Pull request merged**